types/is-odd/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for is-odd 3.0
+// Project: https://github.com/jonschlinkert/is-odd
+// Definitions by:  Adam Zerella <https://github.com/adamzerella>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Return true if a given number is odd or not.
+ */
+declare function isOdd(value: number): boolean;
```

Comment:
From the sample usage, it looks like the library also accepts strings (and anything `Math.Abs` will convert to a number?).

Reply:
@amcasey I wasn't sure how to handle something like this. The function would do its best to convert other types even if it wasn't ideal.

`Math.abs()` handles arrays, objects, strings, null etc... Would it be acceptable to change the definition to something like:

`declare function isOdd(value: number|number[]|string|object|undefined): boolean;`

Reply:
It's a bit of a judgement call.  Personally, I'd support the types shown in the sample usage: `number | string`.